index.d.ts

```diff
@@ -0,0 +1,30 @@
+export interface PromiseWithTime<T> extends Promise<T> {
+	/**
+	The elapsed time in milliseconds.
+	*/
+	readonly time?: number;
+}
+
+declare const pTime: {
+	/**
+	Measure the time a promise takes to resolve.
+
+	@param input - Promise-returning/async function.
```

Comment:
```suggestion
	@param asyncFunction - Promise-returning/async function.
```

?

I'm gonna try to have more descriptive labels when it makes sense.